plugins/statistics.koplugin/main.lua

```diff
@@ -642,13 +615,13 @@ function ReaderStatistics:addBookStatToDB(book_stats, conn)
                 AND    md5 = ?;
         ]]
         local stmt = conn:prepare(sql_stmt)
-        local result = stmt:reset():bind(self.data.title, self.data.authors, self.data.md5):step()
+        local result = stmt:reset():bind(self.data.title, self.data.authors, self.doc_md5):step()
         local nr_id = tonumber(result[1])
         if nr_id == 0 then
             stmt = conn:prepare("INSERT INTO book VALUES(NULL, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?);")
             stmt:reset():bind(book_stats.title, book_stats.authors, book_stats.notes,
                 last_open_book, book_stats.highlights, book_stats.pages,
-                book_stats.series, book_stats.language, self:partialMd5(book_stats.file), total_read_time, total_read_pages) :step()
+                book_stats.series, book_stats.language, util.partialMD5(book_stats.file), total_read_time, total_read_pages) :step()
```

Comment:
Feels like it would be clearer (for debugging/stacktrace purpose) if you had this call above on a standalone line: `local partial_md5 = util.partialMD5(book_stats.file)`, rather than there inline in a complex ljsqlite stuff.